### ISSUE TITLE
documentation bug in `set`

### ISSUE THREAD

**Issue by alan-isaac** [NetLogo/NetLogo#1819]
The [documentation for set)[https://ccl.northwestern.edu/netlogo/docs/dictionary.html#set] refers to "this agent" but provides no supporting example (so "this agent" has no reference in the docs).  Probably an example using `ask` should be included. In any case, the current phrasing is not helpful to newcomers.

**Comment by SethTisue** [comment]
The concept of "this agent" — the currently executing agent — is ambient in NetLogo, it applies to a large number of primitives. In a strong sense, it applies to literally every primitive.

So I don't really understand the suggestion — what is it about `set` in particular...?

**Comment by alan-isaac** [comment]
i. I have a general pedagogical issue with this phrasing, which the dictionary relies on but does not define.  (Of course it poses no problem for experienced users.)
ii. `set` is different because it lacks an example use.

**Comment by ERSUCC** [comment]
I agree that "this agent" is not the best phrasing, especially given that it is never defined. Also, there are a number of primitives in the dictionary that mention "this agent" but do not have the corresponding agent symbols displayed under their name. Either a general terminology section should be added to the dictionary or it should be phrased as "the agent executing this primitive".
